engine/access/rest/routes/router.go

```diff
@@ -230,6 +235,9 @@ func normalizeURL(url string) (string, error) {
 	case 16:
 		// address based resource. e.g. /v1/accounts/1234567890abcdef
 		parts = append(parts, "{address}")
+		if matches[0][5] == "balance" {
```

Comment:
can you add a test for this endpoint as well:
https://github.com/onflow/flow-go/blob/9d73bfbe266403ceeefc52ec14413e37ceb80155/engine/access/rest/routes/router_test.go#L11